packages/bolt-connection/src/channel/deno/deno-host-name-resolver.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { internal } from 'neo4j-driver-core'
+
+const {
+  resolver: { BaseHostNameResolver }
+} = internal
+
+export default class DenoHostNameResolver extends BaseHostNameResolver {
+  resolve (address) {
+    return this._resolveToItself(address)
```

Comment:
It does have, but it is conflicting with the `TLS` configuration. Since the original implementation of the driver-deno was based in the browser version of the driver and this version also resolves to itself, I prefer to go through this route.